acceptance/config/beaker/jenkins/el-6-x86_64-mdca-a.cfg

```diff
@@ -1,25 +1,21 @@
 HOSTS:
-  master:
+  primary:
```

Comment:
nice

Reply:
I assumed that these were arbitrary names and thus changeable, as opposed to the actual roles we set below.

Reply:
Do we actually use these anywhere? I assumed everything is a BHG string for our ci...